pkg/manifests/manifests.go

```diff
@@ -1105,6 +1106,10 @@ func (f *Factory) PrometheusRBACProxySecret() (*v1.Secret, error) {
 	return f.NewSecret(f.assets.MustNewAssetReader(PrometheusRBACProxySecret))
 }
 
+func (f *Factory) PrometheusRBACProxySecretFederate() (*v1.Secret, error) {
```

Comment:
it has to add call to this function here https://github.com/openshift/cluster-monitoring-operator/blob/master/pkg/tasks/prometheus.go#L133-L136
So that the secret will be created by the operator.

Reply:
done